### DISCUSSION TITLE
Customize email subject for "invite user" and "reset password"

### DISCUSSION THREAD

**Discussion by ericlemans** [directus/directus#17598] [Feature Requests]
### Summary

Hi,

I created a custom template for the "invite user" email funcitonality, according to https://docs.directus.io/extensions/email-templates.html

However, I can't find a way to change the default email subject, unless I create a custom endpoint with the whole logic.

Thank you and hope it helps improve the experience!

Eric

### Motivation

I am developing a platform in a different language, and I need to translate the email subject to German, plus add the name of my company.

### Detailed Design

When creating a custom template under /extensions/templates/user-invitation.liquid, I'd like to have the possibility to change the subject text

### Must Have:
- An option to change the email subject

### Drawbacks

The email customization feature is already implemented... and I feel is something many people would be looking forward to having... so can't think of any drawbacks.

### Alternatives

Provide documentation to develop custom endpoints for user invitation. I got the logics from another user in Discord.

### Adoption Strategy

I don't think so... every developer that has implemented this customize email feature, have the same Subject on the email.

**Comment by reckseba** [comment]
Agree. Would be super nice.

Everything seems to be prepared already. "You've been invited" is coded here if subject is not given: here: https://github.com/directus/directus/blob/main/api/src/services/users.ts#L367

subject is not passed here: https://github.com/directus/directus/blob/main/api/src/controllers/users.ts#L296

**Comment by reckseba** [comment]
Same discussion here: https://github.com/directus/directus/discussions/17727 (already with code suggestions)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
